.nais/prod-fss.yml

```diff
@@ -47,7 +47,7 @@ spec:
     - name: JOURNALFORING_URL
       value: https://dokarkiv.nais.adeo.no
     - name: KODEVERK_URL
-      value: http://kodeverk.org.svc.nais.local
+      value: https://kodeverk-api.nav.no
```

Comment:
Denne er vel i GCP?
FSS-gateway brukes bare for tjenester i FSS.
Bør heller gjøre endringen i appen som bruker kodeverk.

Reply:
https://kodeverk-api.nav.no skal slik jeg har skjønt det være ny ingress i alle miljøer, også fss.

Reply:
Jo, men FSS-gateway skal bare inneholde FSS-URL-er.

Det har ikke vært noen kodeverk-kall via FSS-gateway siste 30 dager (ref. [Grafana](https://grafana.nav.cloud.nais.io/d/la_fRB9Mk/selvbetjening-fss-gateway?orgId=1&refresh=5m&from=now-2d&to=now)), så kodeverk-støtten i FSS-gateway kan slettes.

Reply:
Jeg kan fikse det.

Reply:
Da skjønte jeg hva du mente. 🤔